src/constants/lotto.js

```diff
@@ -0,0 +1,27 @@
+export const LOTTO_MESSAGES = Object.freeze({
+  INVALID_LOTTO_PRICE: "[ERROR] 로또 가격은 1000원 단위로 입력해 주세요.",
+  SIX_LENGTH_LOTTO_NUMBER: "[ERROR] 쉼표를 기준으로 6개의 숫자만 입력 가능합니다.",
+  DUPLICATE_LOTTO_NUMBER: "[ERROR] 중복된 숫자가 있습니다.",
+  INVALID_LOTTO_NUMBER: "[ERROR] 1부터 45까지의 숫자만 입력 가능합니다.",
```

Comment:
요기에도 LOTTO_NUMBER를 사용할 수 있지 않을까요?
```suggestion
  INVALID_LOTTO_PRICE: `[ERROR] 로또 가격은 ${LOTTO_NUMBER.DIVISION_PRICE}원 단위로 입력해 주세요.`,
  SIX_LENGTH_LOTTO_NUMBER: `[ERROR] 쉼표를 기준으로 ${LOTTO_NUMBER.LENGTH}개의 숫자만 입력 가능합니다.`,
  DUPLICATE_LOTTO_NUMBER: "[ERROR] 중복된 숫자가 있습니다.",
  INVALID_LOTTO_NUMBER: `[ERROR] ${LOTTO_NUMBER.MIN}부터 ${LOTTO_NUMBER.MAX}까지의 숫자만 입력 가능합니다.`,
```

Reply:
이 부분은 제가 소은님 리뷰에도 비슷한 내용으로 의견을 여쭤보는 리뷰를 남겼는데 지예님도 한번 보시고 의견 남겨주시면 좋을 것 같아요!

상수라는게 변하지 않다는 개념인데 `이렇게 동적인 방식을 사용하는 게 맞나?`, `이렇게 되면 함수나 메서드를 활용해서 해야하지 않나?` 한번 되돌아보게 되더라구요🤔
https://github.com/wooacourse-7th-study/javascript-lotto-6/pull/3#discussion_r1786502252

Reply:
저도 이부분을 되게 많이 고민했는데요. 바꿀 수 없도록 상수로 설정하고 object.freeze를 통해 객체를 동결했는데, .. 이걸 동적인 방식으로 받는게 맞을까 고민했던 것 같아요 .. 이부분은 저도 다시 한번 고민해 봐야될 것 같네요 ..흠..